yt/visualization/plot_window.py

```diff
@@ -306,14 +306,16 @@ def bounds(self):
 
     @invalidate_data
     def zoom(self, factor):
-        r"""This zooms the window by *factor*.
+        r"""This zooms the window by *factor* > 0.
 
         Parameters
         ----------
         factor : float
             multiplier for the current width
 
         """
+        if factor <= 0:
+            raise ValueError("Only positive zooming factors are meaningful.")
```

Comment:
What was your use case for accidentally using negative values in the zoom? I could imagine a user potentially thinking negative values zoom out. This isn't necessary for the PR, but maybe it would be helpful to update our docs on zooming to be explicit that zooming out is defined with values between 0 and 1.

Reply:
Ah nevermind I see you fixed it in the update you pushed while I was reviewing things. Thank you!

Reply:
*Blows out smoking gun* ;-)
And to answer your question I did not really have a use case for trying a zero factor. I just got curious and noticed it was not explicitly forbidden. I tried negative values in an attempt to zoom out and my notebook just... crashed, which motivated the PR :)

Reply:
😎 🆒 🥇

Reply:
Ah I forgot the 🤠 emoji. That's basically perfect for the documentation-sharpshooting you just did. 😉